lib/slimmer/railtie.rb

```diff
@@ -5,10 +5,20 @@ class Railtie < Rails::Railtie
     initializer "slimmer.configure" do |app|
       slimmer_config = app.config.slimmer.to_hash
 
-      app_name = ENV['GOVUK_APP_NAME'] || app.class.parent_name
+      app_name = ENV['GOVUK_APP_NAME'] || module_parent_name
       slimmer_config = slimmer_config.reverse_merge(app_name: app_name)
 
       app.middleware.use Slimmer::App, slimmer_config
     end
+
+  private
+
+    def module_parent_name
```

Comment:
We should remember to remove this when all apps are on Rails 6.

Reply:
Should I add a `TODO`?

Reply:
No, it's fine.

Reply:
Done